08-rails/destroy-action.md

```diff
@@ -0,0 +1,91 @@
+# Intro to the Destroy Action
+
+## Learning Goals
+
+- Understand the responsibility of the Rails convention `destroy` action
+- Review the nominal and edge cases when testing the `destroy` action
+- Use a route parameter in a controller action
+- Learn to make a `delete` request
+
+## The `destroy` Action
+
+The `destroy` action is the last controller action we will define. Its responsibility is, given some information (through route params), find a specific item, and then delete it from the database. **It is the _Delete_ in CRUD.** The `destroy` action is a _Rails convention_, so we will continue to use the name `destroy`.
+
+## Testing
+
+Earlier when we looked at [testing database changes in controllers](`./testing-db-in-controllers.ms`) we examined a test for the `destroy` action.  
+
```

Comment:
This link is incorrect

Reply:
Also, does this mean they'll have seen tests for `destroy` before we get to this lesson? That doesn't feel right to me - I suspect it will be an awkward flow to the week and be confusing to the students.

Looking at [that section of the testing db in controllers lesson](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/08-rails/testing-db-in-controllers.md#content), it might be the fix is to change that to focus exclusively on `create`.

Reply:
Updated